lib/db/dao/activity_topic_dao.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:async';
+
+import 'package:maui/app_database.dart';
+import 'package:maui/db/entity/activity_topic.dart';
+import 'package:sqflite/sqflite.dart';
+
+class ActivityTopicDao {
+  Future<ActivityTopic> getActivityTopic(String id, {Database db}) async {
```

Comment:
This is a join table - which means it is joining activity table with topic table. So there will not be a unique row to return when you query by id. If you want a unique row you should query by activityId and topicId. But I dont think anyone would need such a query. They will probably need a query which will return a list of activities for a given topic. So change this to getActivitiesByTopicId